docs/PORTABLE_STORAGE.md

```diff
@@ -169,9 +169,51 @@ that most will be familiar with):
 }
 ```
 
-This would translate to:
+This object would translate into the following bytes when serialized into epee portable storage format. The bytes are represented in hex, with comments and whitespace added for readability.
 
-![Epee binary storage format example](/docs/images/storage_binary_example.png)
+```
+01 11 01 01 01 01 02 01                         // Signature
+01                                              // Version
+14                                              // Varint size of section (5)
```

Comment:
It's not the size of the section, it's the count of section entries.

Reply:
The rest looks reasonable.

Reply:
Updated PR with your suggestions. Thanks!